src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests_Experimental.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
-    public class AttributeTests_Experimental : CSharpTestBase
+    public class AttributeTests_WindowsExperimental : CSharpTestBase
```

Comment:
Also rename the file?

Reply:
I was intentionally not planning to rename the file. That unfortunately degrades the history and doesn't seem critical for this case.

Reply:
History is only degraded when the changes to the file exceed the threshold of similarity. This one-line change would be detected by Git as a rename and features like blame would not be impacted.

Reply:
Note that the rename can be implemented in a separate commit to ensure Git tracks the rename correctly. The squash-merge anti-feature works hard to undermine core Git functionality, but again this file would not exceed the standard similarity threshold.

Reply:
Do you have an example where blame on GitHub shows the proper history after a file rename?
I've seen git show a rename locally, but never saw blame work.